src/utils/identity.js

```diff
@@ -0,0 +1,3 @@
+export default function identity(value) {
+  return value;
+}
```

Comment:
As @goatslacker suggested, using an arrow function would be better because:

- `identity` is not `new`-able
- `identity` has no `prototype`